extensions/amp-script/0.1/test/validator-amp-script.html

```diff
@@ -87,6 +87,9 @@
   <!-- Invalid: 'development' attribute. -->
   <amp-script layout=container src="https://example.com/foo.js" development></amp-script>
 
+  <!-- Invalid: 'data-ampdevmode' attribute. -->
+  <amp-script layout=container src="https://example.com/foo.js" data-ampdevmode></amp-script>
```

Comment:
Nothing special happens. The fact that `data-ampdevmode` on the root html node skips validation for all other rules is completely special cased in the validator. It doesn't work that way when on other elements.

One more idea though: I can see us wanting dev mode to be implied by putting it on the root as well. I say this because I can't imagine a situation where a publisher would want to suppress all errors on the page, but wouldn't also want to have amp-script in dev mode.

That would mean enabling amp-script's dev mode on either a root html `data-ampdevmode` or on the element itself.

Reply:
Sorry I meant with both `html[data-ampdevmode]` and `amp-script[data-ampdevmode=false]`. I'm guessing it would disable amp-script validation but enforce script hash.

Curious since I think it's been treated as a boolean attribute thus far rather than a value attribute (i.e. `hasAttribute` vs. `getAttribute`).

Reply:
- `html[data-ampdevmode]` is checked via `hasAttribute`. If present, any provided value will invalidate the doc.
- I figured out how to mimick the same functionality via the protoascii configuration (any value, including none, will be a validation error).